pkg/kubelet/volume/gce_pd/gce_util.go

```diff
@@ -139,3 +141,30 @@ func (util *GCEDiskUtil) DetachDisk(pd *gcePersistentDisk, devicePath string) er
 	}
 	return nil
 }
+
+// safe_format_and_mount is a utility script on GCE VMs that probes a persistent disk, and if
+// necessary formats it before mounting it.
+// This eliminates the necesisty to format a PD before it is used with a Pod on GCE.
+// TODO: port this script into Go and use it for all Linux platforms
+type gceSafeFormatAndMount struct {
+	mount.Mounter
```

Comment:
you can embed mount.Interface here instead of mount.Mounter, then Mounter does not have to become public - just initialize it with mount.New().  But this is fine, too.  I was thinking it was cleaner long-term to get rid of mount.New() and just expose Mounter anyway.